### DISCUSSION TITLE
Plugin: How to access ProjectHelper methods in main.js?

### DISCUSSION THREAD

**Discussion by jgulledge19** [TiagoSilvaPereira/vemto-issues#339] [Q&A]
How can I use these methods https://api-docs.vemto.app/ProjectHelper.html is main.js?

**Comment by TiagoSilvaPereira** [comment]
Hi @jgulledge19 do you need these methods inside the template? If yes, you can use **this.projectHelper**

```
this.projectHelper.getModelsNamespace()
```
